samples/language/Bitwise Operators.qs

```diff
@@ -0,0 +1,71 @@
+/// # Sample
+/// Bitwise Operators
+///
+/// # Description
+/// Bitwise operators in Q# perform operations on the bits of integer values,
+/// producing a new integer value. The bitwise operators in Q# are
+/// `~~~`, `&&&`, `|||`, `^^^`, `>>>`, and `<<<`.
+namespace MyQuantumApp {
+
+    @EntryPoint()
+    operation Main() : Unit {
+
+        // `~~~` performs a bit-wise Boolean Not on the bits of an integer.
```

Comment:
Should this be just "bitwise not" instead of "bit-wise Boolean Not". Introducing the "Boolean" word seems confusing since these are not Boolean values nor Boolean variables.

Reply:
done